src/modules/settings/styles/popout.module.css

```diff
@@ -1,12 +1,15 @@
-.default {
-  position: relative;
-  background-color: #292d33;
+.standaloneChatWindow {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  background-color: var(--bttv-rs-bg-overlay);
   height: 100vh;
```

Comment:
instead of setting height and width, let's just set top, left, right, bottom to `0`